akka-docs/rst/scala/http/common/http-model.rst

```diff
@@ -366,4 +366,10 @@ You may also want to read about MediaType `Registration trees`_, in order to reg
 in the right style / place.
 
 .. _Registration trees: https://en.wikipedia.org/wiki/Media_type#Registration_trees
-.. _predefines: https://github.com/akka/akka/blob/master/akka-http-core/src/main/scala/akka/http/scaladsl/model/MediaType.scala#L297
\ No newline at end of file
+.. _predefines: https://github.com/akka/akka/blob/master/akka-http-core/src/main/scala/akka/http/scaladsl/model/MediaType.scala#L297
+
+URI
+------------------------------
```

Comment:
underling must align with length of header, otherwise we get build warnings (which we have 0 of right now :-))